x/wasm/internal/keeper/msg_server.go

```diff
@@ -0,0 +1,185 @@
+package keeper
```

Comment:
Huh, isn't this supposed to be auto-gen code?

Reply:
I guess not: https://github.com/cosmos/cosmos-sdk/blob/master/x/bank/keeper/msg_server.go